common/src/stack/command/stack/commands/report/host/__init__.py

```diff
@@ -11,9 +11,12 @@
 # @rocks@
 
 import stack.commands
+from stack.commands import Log
+from stack.exception import CommandError
 import stack.text
 import os.path
 import shlex
+import syslog
```

Comment:
Would rather see a new function "Warn()" defined just below `Log()` that did the syslog stuff so every file that wants to Warn doesn't need to import this.